lib/plugins/aws/package/compile/events/s3/index.js

```diff
@@ -21,7 +21,11 @@ class AwsCompileS3Events {
 
   newS3Buckets() {
     const bucketsLambdaConfigurations = {};
+    const bucketsLogicalIds = {};
     const s3EnabledFunctions = [];
+    const resources =
```

Comment:
Let's name it `userResources` (to not confuse with resource data that's generated by the framework)